src/Content/ContactSelector.php

```diff
@@ -105,12 +105,13 @@ private static function getServerURLForProfile($profile)
 	}
 
 	/**
-	 * @param string $network network
-	 * @param string $profile optional, default empty
+	 * @param string $network  network
```

Comment:
Please explicit the difference between "network" and "protocol" and the expected data source in both. Both the "optional, default empty" are inferred through the function prototype and can be removed.